documentation_quality_analysis/analyze_library/doc_parser/doc_signature_util.py

```diff
@@ -17,29 +17,76 @@ def get_signatures_from_doc(doc_page: DocPage) -> List[Signature]:
     soup = doc_page.content
     signatures: List[Signature] = []
 
-    signatures.extend(get_signatures_from_section(soup, PARSE_GENERATED_HTML))
+    dt_tags = soup.find_all("dt")
+
+    if dt_tags:
+        parsing_method = PARSE_GENERATED_HTML
+        signatures.extend(get_signature_from_generated_html(doc_page))
+
+    else:
+        parsing_method = PARSE_ANY_DOC
```

Comment:
what's the point of setting `parsing_method` if it's not being used anywhere (as far as I can tell)?